website/source/layouts/github.erb

```diff
@@ -16,6 +16,12 @@
 					<li<%= sidebar_current("docs-github-resource-membership") %>>
 					<a href="/docs/providers/github/r/membership.html">github_membership</a>
 					</li>
+					<li<%= sidebar_current("docs-github-resource-team-membership") %>>
+                    <a href="/docs/providers/github/r/team_membership.html">github_team_membership</a>
+                    </li>
```

Comment:
Yeah I'm not sure what's going on there, I did a spaces->tabs reformat with intellij on it and didn't seem to help but I'll try something else